packages/extension-bubble-menu/src/bubble-menu-plugin.ts

```diff
@@ -3,7 +3,7 @@ import {
 } from '@tiptap/core'
 import { EditorState, Plugin, PluginKey } from '@tiptap/pm/state'
 import { EditorView } from '@tiptap/pm/view'
-import debounce from 'lodash/debounce'
```

Comment:
ig using `import debounce from 'lodash/debounce.js'` also fixes it

Reply:
asking @bdbch again since I didn't get any response, have you tried this solution? if this solution works, everyone has `lodash` installed in their projects so they don't need to add a new package and using `lodash-es` it'll increase the build size